parler/models.py

```diff
@@ -1085,6 +1187,7 @@ def __init__(self, base, shared_model, translations_model, related_name):
 
         self.base = base
         self.inherited = False
+        self._local_extensions = []
```

Comment:
What is "local extensions" doing? I find the name a bit vague

Reply:
It needs to know which extensions are belong to model or come from parent model through model inheritance.
We create additional attributes only for new extensions. Agree, the name is not clear. I was following the logic we have in django's model._meta: we have `fields`, `concrete_fields`, `many_to_many` attributes to access to all fields of the model and we have `local_fields`, `local_concrete_fields`, `local_many_to_many` to access only to current model fields without inherited fields.